extensions/kotlin/runtime/pom.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.quarkus</groupId>
+        <artifactId>quarkus-kotlin-parent</artifactId>
+        <version>999-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>quarkus-kotlin</artifactId>
+    <name>Quarkus - Kotlin</name>
```

Comment:
Minor nitpick, sorry for not spotting this earlier, I think all the others are suffixed ` - Runtime`. Not sure if we will keep it but better be consistent for now.

Reply:
Good catch! Fixed